src/redux/sagas/extensions/extensionInstall.ts

```diff
@@ -132,7 +132,7 @@ function* handleMultiSigInstall(colonyAddress: string, metaId: string) {
           childSkillIndex,
           userId,
           Id.RootDomain,
-          intArrayToBytes32([1]),
+          intArrayToBytes32([1, 2, 3, 4, 5, 6]),
```

Comment:
4 is unnecessary in this as it is unused (I think this whole chunk will need refactoring to apply all users existing permissions as multi-sig permissions anyway, but will open a new issue for this)